plugin.video.nhlgcl/addon.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2016.11.18" provider-name="eracknaphobia">
+<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2017.1.24" provider-name="eracknaphobia">
   <requires>
    <import addon="xbmc.python" version="2.19.0"/>   
    <import addon="script.module.simplejson" version="3.3.0"/>  
    <import addon="script.module.pytz" version="2014.2" />   
-   <import addon="script.module.beautifulsoup4" version="4.3.1" />  
+   <import addon="script.module.beautifulsoup4" version="4.3.1" /> 
+   <import addon="script.module.pil" version="1.1.7" /> 
   </requires>
   <extension point="xbmc.python.pluginsource" library="main.py">
         <provides>video</provides>
   </extension>  
   <extension point="xbmc.addon.metadata">    
     <platform>all</platform>
-    <summary lang="en">Watch out-of-market regular season games of every team LIVE from around the league</summary>
-    <description lang="en">Watch out-of-market regular season games of every team LIVE from around the league</description>
+    <summary lang="en">Watch out-of-market games from around the League on Kodi, with HD quality streaming</summary>
+    <description lang="en">With NHL.TV you can access revolutionary 60fps (frames per second) video through Kodi</description>
     <disclaimer lang="en"></disclaimer>
     <language>en</language>
```

Comment:
JFYI you can also use the news tag when you update a plugin (in addition to the changelog)